### ISSUE TITLE
General selector is overridden

### ISSUE THREAD

**Issue by jakubzaba** [kiwicom/react-kiwi-ui#30]
Multiple component on the same page causes style is overridden.
**Selector should be unique / generated by styled-jsx.**

E.g. `Typography` style is defined
```html
<style jsx>{`
    span {
      font-family: Roboto, -apple-system, sans-serif;
      font-size: ${sizes[props.size]};
      color: ${colors[props.type]};
    }
`}</style>
```
So last with type `<Typography type="primary">` will make all others `span` with the `primary` color.
<img width="356" alt="screen shot 2018-02-26 at 4 18 10 pm" src="https://user-images.githubusercontent.com/1803950/36678313-b107907e-1b10-11e8-9a2e-1b9784c116ea.png">

----

Make sure these boxes are ticked before submitting a new issue:

* [x] I have checked that this is not a duplicate
* [x] I have added a label categorizing this under BLKN | Kiwicom | Icons | Global

**Comment by jakubzaba** [comment]
Sorry, my bad. It should work with general selector, but it's not transpiled when I'm using it in BLKN.

**Comment by Fallion** [comment]
Should be closed by #40.

@jakubzaba let me know if there are still issues.
